packages/image/rollup.config.js

```diff
@@ -4,7 +4,7 @@ import pkg from './package.json';
 
 export default {
   input: 'src/index.js',
-  external: Object.keys(pkg.dependencies),
+  external: [...Object.keys(pkg.dependencies), ...['fs', 'path']],
```

Comment:
For [other plugins](https://github.com/rollup/plugins/blob/master/packages/inject/rollup.config.js#L5) we've followed the format of having `external` be a separate variable.

```js
const external = Object.keys(pkg.dependencies).concat('fs', 'path');
```

Reply:
there's functionally no difference there, and really no readability difference either. I don't think this is a spot we need to enforce a pattern.

Reply:
In that case let's remove the unnessecary spread:
```suggestion
  external: [...Object.keys(pkg.dependencies), 'fs', 'path'],
```

Reply:
That's a neat trick. And this whole time I thought I had to spread anything behind a spread.